### DISCUSSION TITLE
similar projects

### DISCUSSION THREAD

**Discussion by cauerego** [kcubeterm/achoz#23] [Ideas]
of course, i don't expect any to use agpl+ (as every project should)

**Comment by cauerego** [comment]
https://github.com/jina-ai/jina/

ai artificial intelligence simplified.

might be cloud only.

**Comment by cauerego** [comment]
http://apse.io

hard to grasp exactly what it does and how.

is it a desktop-only app that captures the screen and use computer vision to sort and index everything out?

**Reply by kcubeterm** [reply]
It is not opensource oops, but promising

**Reply by cauerego** [reply]
i wouldn't be surprised if it's windows only.
